src/plugins/telemetry/telemetry_impl.cpp

```diff
@@ -2255,52 +2473,134 @@ void TelemetryImpl::check_calibration()
                     std::placeholders::_2),
                 this);
         }
-    } else {
-        _parent->remove_call_every(_calibration_cookie);
```

Comment:
Is this intentional?

Reply:
Yes, as far as I understand this is called once we no longer want to keep calling this check_calibration(). Before we wanted to not check calibration when our system was detected as Ardupilot and thus we remove it the first time check_calibration gets called. Now we do want to keep checking calibration until we have gotten healthy values, hence this is removed.